### PR TITLE
`Assessment`: Improve description of complaints

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

Artemis only allows one complaint per exercise. Some students were confused why they could not write another complaint after sending the first one, even though they still had tokens left.

### Description

This PR improves the description of complaints shown to students to clearly state that only one complaint per exercise is possible. The documentation of the complaints is also improved and states this clearly now.

### Steps for Testing

1. Check the new strings in the .json and .rst-files

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

File: src/main/webapp/i18n/en/complaint.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
             "info": "The complaint limit refers to open and rejected complaints. If your complaint is accepted it will not count towards this total.",
             "description": "You may submit up to {{ maxComplaintNumber }} complaints in this course.",
             "descriptionExam": "You can submit one complaint for each manually assessed exercise in this exam.",
-            "descriptionExtended": "Every student is allowed to submit up to {{ maxComplaintNumber }} complaints in this course. You still have <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
+            "descriptionExtended": "Every student is allowed to complain once on an exercise. In total {{ maxComplaintNumber }} complaints are possible in this course. You still have <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/en/complaint.json:11]:
```suggestion
            "descriptionExtended": "Every student is allowed to complain once per exercise. In total {{ maxComplaintNumber }} complaints are possible in this course. You still have <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
```

**Review by Strohgelaender — APPROVED** [review]

**Review by b-fein — DISMISSED** [review]

File: src/main/webapp/i18n/en/complaint.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
             "descriptionTeam": "Your team may submit up to {{ maxComplaintNumber }} complaints in this course.",
             "descriptionTeamExtended": "Every team is allowed to submit up to {{ maxComplaintNumber }} complaints in this course. Your team still has <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
```
Comment (mallgaier) [inline src/main/webapp/i18n/en/complaint.json:13]: _"descriptionTeamExtended": "One student per team can complain once on an exercise. A total of up to {{ maxComplaintNumber }} complaints per team can be submitted in this course. [...]_

**Review by mallgaier — DISMISSED** [review]
Overall, a good, understandable change. 
I have left suggestions, on how the description for Complaints about Team-Exercises could be improved in my opinion.  If this is outside of the scope of this PA, please ignore.

**Review by cremertim — APPROVED** [review]
lgtm

**Review by Strohgelaender — APPROVED** [review]

File: src/main/webapp/i18n/de/complaint.json [hunk 3]
```diff
@@ -8,9 +8,9 @@
             "info": "Die Limitierung der Beschwerdenanzahl bezieht sich auf offene und abgelehnte Beschwerden. Wenn deine Beschwerde angenommen wird, wird sie nicht auf deine Gesamtzahl der Beschwerden angerechnet.",
             "description": "Du kannst insgesamt bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs einreichen.",
             "descriptionExam": "Du kannst zu jeder manuell korrigierten Aufgabe der Klausur eine Beschwerde einreichen.",
-            "descriptionExtended": "Alle Studierenden dürfen jeweils insgesamt bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs einreichen. Du hast noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
+            "descriptionExtended": "Alle Studierenden dürfen genau eine Beschwerde für eine Aufgabe einreichen. Insgesamt sind bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs möglich. Du hast noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
```
Comment (julian-christl) [inline src/main/webapp/i18n/de/complaint.json:11]:
```suggestion
            "descriptionExtended": "Alle Studierenden dürfen genau eine Beschwerde für eine Aufgabe einreichen. Insgesamt sind {{ maxComplaintNumber }} Beschwerden in diesem Kurs möglich. Du hast noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
```

File: src/main/webapp/i18n/de/complaint.json [hunk 4]
```diff
@@ -8,9 +8,9 @@
...
             "descriptionTeam": "Ihr könnt insgesamt bis zu {{ maxComplaintNumber }} Beschwerden als Team in diesem Kurs einreichen.",
-            "descriptionTeamExtended": "Jedes Team darf insgesamt bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs einreichen. Ihr habt noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
+            "descriptionTeamExtended": "Jedes Team darf genau eine Beschwerde für eine Aufgabe einreichen. Insgesamt sind bis zu {{ maxComplaintNumber }} Beschwerden in diesem Kurs möglich. Ihr habt noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
```
Comment (julian-christl) [inline src/main/webapp/i18n/de/complaint.json:13]:
```suggestion
            "descriptionTeamExtended": "Jedes Team darf genau eine Beschwerde für eine Aufgabe einreichen. Insgesamt sind {{ maxComplaintNumber }} Beschwerden in diesem Kurs möglich. Ihr habt noch <strong style=color:red;>{{ allowedComplaints }}</strong> Beschwerden übrig.",
```

**Review by julian-christl — APPROVED** [review]
<3

**Review by Strohgelaender — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by mallgaier — APPROVED** [review]
Looks great!

**Review by krusche — APPROVED** [review]

**Pull request merged**